packages/contracts-bedrock/test/L2/L1Block.t.sol

```diff
@@ -57,7 +65,42 @@ contract L1BlockBedrock_Test is L1BlockTest {
             _sequenceNumber: type(uint64).max,
             _batcherHash: bytes32(type(uint256).max),
             _l1FeeOverhead: type(uint256).max,
-            _l1FeeScalar: type(uint256).max
+            _l1FeeScalar: type(uint256).max,
+            _dependencySet: _dependencySet
+        });
+    }
+
+    /// @dev Tests that `setL1BlockValues` fails if sender address is not the depositor
+    function test_setL1BlockValues_notDepositor_fails(uint256[] calldata _dependencySet) external {
+        vm.assume(_dependencySet.length <= type(uint8).max);
+        vm.expectRevert("L1Block: only the depositor account can set L1 block values");
+        l1Block.setL1BlockValues({
+            _number: type(uint64).max,
+            _timestamp: type(uint64).max,
+            _basefee: type(uint256).max,
+            _hash: keccak256(abi.encode(1)),
+            _sequenceNumber: type(uint64).max,
+            _batcherHash: bytes32(type(uint256).max),
+            _l1FeeOverhead: type(uint256).max,
+            _l1FeeScalar: type(uint256).max,
+            _dependencySet: _dependencySet
+        });
+    }
+
+    /// @dev Tests that `setL1BlockValues` fails if the length of the dependency set is greater than uint8
+    function test_setL1BlockValues_invalidDependencySetLength_fails(uint256[] calldata _dependencySet) external {
+        vm.assume(_dependencySet.length > type(uint8).max);
+        vm.prank(depositor);
```

Comment:
I don't see a `vm.expectRevert`?

Reply:
This test should be moved to the interop interface on the L1Block contract

Reply:
this test is unnecessary since the argument `dependencySet` not longer exists in `setL1BlockValues`